aries-site/public/extend-preview-chomp.svg

```diff
@@ -0,0 +1,14 @@
+<svg width="336" height="192" viewBox="0 0 336 192" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Thinking we should move all of these images into a subdirectory like `public/images/card-previews` or something along those lines. Having them all at the public root is a little overwhelming.

Reply:
lol sounds good to me it is a little much!